itou/templates/layout/base.html

```diff
@@ -64,6 +64,15 @@
         <div class="global-messages-container">
             {% block global_messages %}
                 {% include "includes/demo_accounts.html" %}
+                {# TODO: when deleting this banner, delete Company.has_job_descriptions_not_updated_for too #}
+                {% if request.user.is_employer and request.current_organization and request.current_organization.has_job_descriptions_not_updated_for %}
```

Comment:
Peut-être petit problème de nommage pour `.has_job_descriptions_not_updated_for()`, car quand on lit le template on ne peux que se demander "for quoi ?" :).
Proposition alternative : _has_job_descriptions_not_updated_recently_, voir _has_not_updated_job_descriptions_recently_.